torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -204,6 +211,8 @@ class AutoAugment(_AutoAugmentBase):
         "ShearY": (lambda num_bins, image_size: torch.linspace(0.0, 0.3, num_bins), True),
         "TranslateX": (lambda num_bins, image_size: torch.linspace(0.0, 150.0 / 331.0 * image_size[1], num_bins), True),
         "TranslateY": (lambda num_bins, image_size: torch.linspace(0.0, 150.0 / 331.0 * image_size[0], num_bins), True),
+        # How do you feel about explicitly passing height, width here instead of image_size? I did it in an earlier PR
+        # but removed it due to the verbosity. Thoughts?
```

Comment:
Originally I stated I also prefer the non-verbose way that we currently have. Thinking about it more, the extra verbosity wouldn't hurt here since this is not user facing and would make the instances where we actually need the height and width more clear. Thus, I'm now in slight favor of passing height and width separately.

Reply:
I've literally introduced this and took it back on subsequent commits (see [here](https://github.com/pytorch/vision/pull/5487/commits/ead34b3ea23ae207f3667569fe3ce22c42b3c841)). I'm OK if we be verbose.